config.py

```diff
@@ -0,0 +1,31 @@
+"""
+STATIC and SECRET data
+"""
+from pydantic import BaseSettings
+
+from functools import lru_cache
+
+
+class Settings(BaseSettings):
+    MAIL_USERNAME: str
+    MAIL_PASSWORD: str
+    MAIL_FROM: str
+    MAIL_PORT: int
+    MAIL_SERVER: str
+    MAIL_TLS: bool
+    MAIL_SSL: bool
+    USE_CREDENTIALS: bool
+    VALIDATE_CERTS: bool
+    SECRET: str
+    SITE_URL: str
+    SITE_NAME: str
+
+    class Config:
```

Comment:
**Static Code Issue**: Missing docstring in public nested class Config. <br> **Fix**: Add a docstring to the nested Config class. <br> **Code Suggestion**: 
 ```
 +        """Configuration for settings class, specifying the environment file location."""
 ```